File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToAuthRequestConverter.java
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request;

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.AuthRequest;
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel;
import org.springframework.core.convert.converter.Converter;
import org.springframework.stereotype.Component;

@Component
public class EntryStateToAuthRequestConverter implements Converter<CustomEntryStateModel, AuthRequest> {

    @Override
    public AuthRequest convert(CustomEntryStateModel entryStateModel) {
        return new AuthRequest();
    }
}

File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToAuthRequestConverter.kt
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.AuthRequest
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel
import org.springframework.core.convert.converter.Converter
import org.springframework.stereotype.Component

@Component
class EntryStateToAuthRequestConverter : Converter<CustomEntryStateModel, AuthRequest> {

    override fun convert(entryStateModel: CustomEntryStateModel): AuthRequest {
        return AuthRequest()
    }
}
